src/common/transformations/tests/utils/convert_precision.cpp

```diff
@@ -1787,3 +1793,119 @@ TEST(TransformationTests, ConvertPrecision_exp_through_unsqueeze) {
     FunctionsComparator::Result result = func_comparator(model_ref, model);
     ASSERT_TRUE(result.valid) << result.message;
 }
+
+TEST(TransformationTests, ConvertPrecision_disable_for_quantized_nodes_1) {
+    shared_ptr<Model> model, model_ref;
```

Comment:
I agree that `TEST_F` would be shorter. However this will require additional time to correct other test-cases for `ConvertPrecision` since in a single file `TEST` and `TEST_F` suites should not be mixed. 

Can we skip this improvement for now and do it next time when the current solution will be improved to support general case?

Reply:
Agree